src/banana/utils.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pathlib
+from contextlib import contextmanager
+
+import pytest
```

Comment:
Please only move `lhapdf_path` here - all the rest is more stupid and truely test related

Reply:
Since we want to use `banana` for more serious projects, I would skip the dependency on `pytest`, we can accept to reduce coverage (hopefully at some point `lhapdf` will be a PyPI package :weary:)